docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -0,0 +1,268 @@
+    insert into employees (employee_no, name, department, salary) values (1222, 'Bette Davis', 'Sales', 55000);
+    insert into employees (employee_no, name, department, salary) values (1223, 'Lucille Ball', 'Operations', 70000);
+    insert into employees (employee_no, name, department, salary) values (1224, 'John Zimmerman', 'Sales', 60000);
+
+    SELECT * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+1. Create a snapshot of the table from a shell prompt:
+
+    ```sh
+    $ bin/yb-admin create_database_snapshot pitr employees
+    ```
+
+    ```output
+    Started snapshot creation: bb5fc435-a2b9-4f3a-a510-0bacc6aebccf
+    ```
+
+1. Verify that the snapshot is complete:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+### Restore from an absolute time
+
+1. Get a timestamp. YCQL doesn't have a now() function, so use a command such as one of the following:
+
+    ```sh
+    # Ruby: remove the decimal point before using the timestamp
+    $ ruby -e 'puts Time.now.to_f'
+    ```
+
+    ```sh
+    # Python: remove the decimal point before using the timestamp
+    $ python -c 'import datetime; print datetime.datetime.now().strftime("%s.%f")'
+    ```
+
+    ```sh
+    # Linux and some other systems (but not macOS): use the timestamp as-is
+    $ date +%s%N | cut -b1-15
+    ```
+
+1. Add a row for employee 9999 to the table:
+
+    ```sql
+    insert into employees (employee_no, name, department, salary) values (9999, 'Wrong Name', 'Marketing', 10000);
+
+    select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            9999 |     Wrong Name |  Marketing |  10000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (5 rows)
+    ```
+
+1. List snapshots, at a terminal prompt:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+1. Restore the latest snapshot to the timestamp you obtained before you deleted the data, at a terminal prompt:
+
+    ```sh
+    $ bin/yb-admin restore_snapshot bb5fc435-a2b9-4f3a-a510-0bacc6aebccf 1617670679185100
```

Comment:
The timestamp '1617670679185100' doesn't seem to have a source.  Maybe you should use that as the example output somewhere in the 'Get a timestamp' section.

Reply:
Added output samples for the various shell options, and grabbed one for this example. Thanks!